cmd/search.go

```diff
@@ -51,6 +51,29 @@ func bail(err error) {
 	os.Exit(1)
 }
 
+func completion(cmd *cobra.Command, c string) {
+	switch c {
+	case "bash":
+		err := cmd.GenBashCompletion(os.Stdout)
+		if err != nil {
+			bail(fmt.Errorf("Failed to generate bash completion: %w", err))
+		}
+	case "zsh":
+		if err := cmd.GenZshCompletion(os.Stdout); err != nil {
+			bail(fmt.Errorf("Failed to generate zsh completion: %w", err))
+		}
+	case "fish":
+		if err := cmd.GenFishCompletion(os.Stdout, true); err != nil {
+			bail(fmt.Errorf("Failed to generate fish completion: %w", err))
+		}
+	case "powershell":
```

Comment:
We should add a default case that bails and errors out if someone passes in garbage like "pwrshl", or an unsupported shell like "ksh".